spec/rails_helper.rb

```diff
@@ -88,6 +88,19 @@ def self.capybara_tmp_path
   end
 end
 
+module RSpec
+  module Core
+    def self.skipped_last
+      @@skipped_last ||= false
+    end
+
+    def self.skipped_last=(value)
+      @@skipped_last = value
+    end
+  end
+end
+
+
```

Comment:
This is all I could come up with to persist metadata between `before(:all)` hooks.

Reply:
Have you tried using `Thread.current[:skipped_last_test]`?

Reply:
Had no clue about that. It does in fact work.